policy/modules/xen/uivm.te

```diff
@@ -45,3 +45,5 @@ ndvm_send_argo(uivm_t)
 iomem_map_read_mmu(uivm_t)
 xen_write_console(uivm_t)
 dom0_copy_grant(uivm_t)
+argo_register_any_source(uivm_t)
```

Comment:
Does the UIVM need permission to register any single-source rings? The any-source rings would be for services listening for incoming connections -- and a common response to that in a program using sockets is to establish a single-source ring for handling the connection, hence the question.

Reply:
IIRC the UIVM uses an any-ring to handle the UID stuff, but I'm not 100% on that. My guess is yes since there were no denials that suggested otherwise.

Reply:
Never mind, you're right. I can't sshargo without it.